src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3056,6 +3056,43 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/snapshotAccessRequests/{id}:
+    get:
+      tags:
+        - SnapshotAccessRequest
+      description: Get a snapshot access request you have access to read
+      operationId: getSnapshotAccessRequest
+      parameters:
+        - $ref: '#/components/parameters/Id'
+      responses:
+        200:
+          description: The request fetched
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SnapshotAccessRequestResponse'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
```

Comment:
I know we've talked about improving our error handling, so I don't want to go too crazy here, but I think we could specifically call out the 404 option where the request is not found.

Reply:
Sounds reasonable, especially given we explicitly _can_ throw that.